introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5.ipynb

```diff
@@ -934,6 +934,115 @@
     "    print(f\"{bold} The reasoning result is{unbold}: '{generated_texts}'{newline}\")"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        "top_p": 0.95,
</pre><p>Please check how top_p and top_k work together. I believe one supersedes the other, and specifying both together does not help.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>This has been answered by Kyle in the science chat room</p>